manager/controllers/app/storage.go

```diff
@@ -37,15 +37,23 @@ func AllocateBucket(c client.Client, log logr.Logger, owner types.NamespacedName
 		log.Info(err.Error())
 		return nil, err
 	}
+
 	for _, account := range accountList.Items {
 		utils.PrintStructure(account, log, "Account ")
-		if !includesGeography(account.Spec.Regions, geo) {
```

Comment:
After you go over endpoint+region pairs, select one pair that matches region=geo and pass the relevant endpoint to the ProvisionedBucket structure.

Reply:
Changed